src/DependencyInjection/Configuration.php

```diff
@@ -118,6 +119,7 @@ private function createClientsNode()
                     ->arrayNode('middleware')
                         ->prototype('scalar')->end()
                     ->end()
+                    ->append($this->createSerializerNode())
```

Comment:
Do we actually need a specific serializer for different clients ?

Regardless, generally the configuration / mapping is still the same ?